index.bs

```diff
@@ -491,6 +491,13 @@ When requested, the {{XRSession}} MUST <dfn>apply pending render states</dfn> by
       <dt> Otherwise
       <dd> Set |activeState|'s [=XRRenderState/output canvas=] to <code>null</code>.
     </dl>
+  1. Set |activeState|'s [=XRRenderState/direct output=] boolean based on the following:
```

Comment:
Can this section be combined with the one above?  Or at the very least the switch be based on whether the "output canvas" property is null?

Reply:
Yeah, we can do that. This will need to be torn apart when we get back to more advanced canvas ouput, but that's a problem for future me. :)